tests/parser/functions/test_extract32.py

```diff
@@ -20,7 +20,6 @@ def extrakt32_storage(index: int128, inp: Bytes[100]) -> bytes32:
     test_cases = (
         (b"c" * 31, 0),
         (b"c" * 32, 0),
-        (b"c" * 32, -1),
```

Comment:
Why was this test case removed? Because of the type change above?

Reply:
Yes, for the same reasons above.